docs/openapi3.yaml

```diff
@@ -323,6 +322,40 @@ paths:
               $ref: '#/components/schemas/benchmark_request'
         description: The bench mark to add
         required: true
+    get:
+      tags:
+        - Tests
+      parameters:
+        - in: path
+          name: test_id
+          description: The test id.
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 4bf5d7ab-f310-4a64-8ec2-d65c06188ec1
+      summary: get a bench mark for test
+      description: Create a bench mark for test
```

Comment:
description: Create a bench mark for test 
change
description: Get a benchmark for a test

Reply:
fixed